src/index.ts

```diff
@@ -356,6 +357,23 @@ function ${cls.name}${
 		});
 	}
 
+	let static_fields = "";
+	if (cls.static_properties !== undefined) {
+		static_fields += " ";
+
+		const length = cls.static_properties.length - 1;
+		cls.static_properties.forEach((prop, index) => {
+			static_fields += `${prop.name} = ${prop.value}`;
+			
+			if (index < length)
+			{
+				static_fields += ", ";
+			}
```

Comment:
This is a very imperative way of joining the table elements together. It'd be a bit more idiomatic to instead do something like this (pseudocode):
```ts
const staticFields = cls.static_properties?.map((...) => `foo = bar`).join(", ") ?? "";
```

⬆️ This will produce `{key = value}` instead of `{ key = value }` as your code currently does, but that seems fine given the output doesn't need to be well formatted?